setup.cfg

```diff
@@ -1,4 +1,47 @@
-[egg_info]
-tag_build = .dev0
-#tag_svn_revision = yes
-#tag_date = yes
+[metadata]
+name = dill
+author = Mike McKerns
+maintainer = Mike McKerns
+license = BSD-3-Clause
+license_file = LICENSE
+description = serialize all of python
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://pypi.org/project/dill
+homepage = https://github.com/uqfoundation/dill
+download_url = https://github.com/uqfoundation/dill
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.5
+    Programming Language :: Python :: 2.6
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.1
+    Programming Language :: Python :: 3.2
+    Programming Language :: Python :: 3.3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Topic :: Scientific/Engineering
+    Topic :: Software Development
+platforms = Linux, Windows, Mac
+
+[options]
+packages = dill, dill.tools
+zip_safe = False
+python_requires = >=2.5, !=3.0.*
+setup_requires = setuptools; wheel; setuptools_scm
```

Comment:
While, in the next release or so, I will be trimming off support for some older versions of python, using `setuptools_scm` will force a drop of several old versions.  Old versions need to be supported longer in `dill` as compared to other packages, as people need a way to open old stored pickled object files.